### ISSUE TITLE
Figure out better placement for error state on Connection Management screen 

### ISSUE THREAD

**Issue by morebrownies** [OpenBazaar/openbazaar-desktop#312]
![screenshot 2017-01-13 16 08 21](https://cloud.githubusercontent.com/assets/13066886/21949872/884a5ea8-d9aa-11e6-992e-20b1882efb03.png)

**Comment by tenthhuman** [comment]
let's cut things out until it's minimal

key point is we can reduce copy to one word

fail

and move this new one word message away from the common area at the top and onto the connection that fails

the fail word can when clicked expand the failed connection item height to encompass a more detailed message and information to remedy

alternately on hover a hover card anchored to the word fail floats above with that info

alternately instead of word fail being the target a question mark positioned next to could be. I prefer just the word as it's simpler

I prefer expanding items to hover cards in this case

styling fail can be like the copied notification styling on donation tab in about modal with color a red instead of a black.

everything else is unchanged by this

some arguments for this pattern follow

- perfection is when there's nothing left to take away
- when something fails on Facebook it's minimal indication that expands
- the current pattern while effective is not a familiar pattern
- the proposed pattern is less busy
- the only thing that can go wrong with a single connection is: it didn't connect. people understand connected vs not connected. no need to make two sentences where one word conveys
- if different conditions mandate different remedies these can hide behind the expand action
- if an error state presents that is not due to individual connections failing but to something more general then this cross cutting state can be indicated in the common area at top concisely.
 in that case no red left border, which looks great on the items and does not work as well up top. instead a word like 

offline 

in a red border offset from its parent box by padding or margin. 

which again could expand into detail

to summarise

- nothing but necessaries
- hide details behind interactions
- visually group error info with its source

**Comment by morebrownies** [comment]
> when something fails on Facebook it's minimal indication that expands

Facebook users don't have to deal with complicated server configurations. This has been one of the biggest friction points of OpenBazaar 1.0 and adding some more detail in the error responses is something we wanted to try to incorporate. More is probably better than less in this scenario.

**Comment by morebrownies** [comment]
@rmisio here are the updated designs for improving the error state. There may be some cases where the error extends to 2 lines, but I don't think it will be too much of a problem. 

![screenshot 2017-01-31 13 53 09](https://cloud.githubusercontent.com/assets/13066886/22486012/9f293aac-e7bc-11e6-874f-ff7e34b2cb3c.png)

![screenshot 2017-01-31 13 52 59](https://cloud.githubusercontent.com/assets/13066886/22486014/9f3294d0-e7bc-11e6-8c81-728896bf2dad.png)

![screenshot 2017-01-31 13 52 51](https://cloud.githubusercontent.com/assets/13066886/22486013/9f2e7544-e7bc-11e6-8d45-d41622d0c9d7.png)

**Comment by jjeffryes** [comment]
@morebrownies do you still want to make the design changes here?

**Comment by rmisio** [comment]
I vote the statute of limitations has run out.
